LanguageFeatures/Static-access-shorthand/equality_A02_t08.dart

```diff
@@ -28,22 +28,22 @@ extension type const ET<T>(T value) {
 }
 
 main() {
-  if ([ET(0)] == const [.new(0)]) {}
+  if ([ET(0)] case == const [ET.new(0)]) {}
```

Comment:
Wouldn't this need a similar treatment as the other tests? (That is, the `.new` is not changed to `ET.new`, and the caret is moved to the `.`).

Same thing in line 36, 41, 46.

Reply:
Added `ET.` to check that there were no bugs and accidentally didn't remove it, sorry. Fixed now.